base58.go

```diff
@@ -1,18 +1,22 @@
 package coincodec
 
 import (
-	"bytes"
-	"crypto/sha256"
 	"fmt"
 	"math/big"
 	"strings"
 )
 
+const (
+	// Default alphabet, used e.g. by Bitcoin
+	Base58DefaultAlphabet = 
+	   "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz"
```

Comment:
since we are here, let's remove reference to `btcutil/base58`? add `https://github.com/btcsuite/btcutil/blob/master/base58/alphabet.go` and xrp alphabet for better performance